### ISSUE TITLE
Update list of MPD clients 

### ISSUE THREAD

**Issue by kingosticks** [mopidy/mopidy#1171]
The MPD documentation could do with an update. Specifically:
- [x] Retesting with more recent versions
- [x] Remove PMix (no longer available)
- [x] Remove Droid MPD Client (last updated 2011)
- [ ] ~Add Remotedy~
- [ ] Add link to the play store search results for 'MPD'
- [ ] Mention effects of `command_blacklist` for any specific clients

**Comment by hkultraman** [comment]
BTW, I have tested "Remotedy" 
( https://play.google.com/store/apps/details?id=se.anil.remotedy )

Which is very good,with good UI and fast UI response. See if anyone can help to update the MPD client list.

**Comment by rawdlite** [comment]
It tested a few Clients with Mopidy 1.0.5
This is a prelimary overview that i hope to validate and bring in to a table.

MPDroid 1.07.2
Connection: Only one Server can be configured
Library Browsing: Artist and Album broken (worked with previous mopidy version)
Streaming Backends:  Bassdrive, SomaFM, Soundcloud, Spotify browsable
Playlist: Spotify Playlist are shown flat.
Search: Simple Search works partly. Only Album and Title
(Artist detailing fails with Client sent us blacklisted command: listallinfo).

MPD Control 1.20 (14)
Connection: Only one Server can be configured
Library Browsing: not working
Streaming Backends:  Bassdrive, SomaFM, Soundcloud, Spotify browsable
Playlist: Spotify Playlist are shown flat.
Search: No Search

Remotedy 1.0.13
Connection: Multiple Servers van be configured (uses http api)
Library Browsing:  not supported
Streaming Backends:  Browsing not supported
Playlist: Spotify Playlist are shown flat.
Search: Simple Search works

None of these Clients allows to limit the search to specific tags like Droid MPD Client did.
No Client allows to limit the search to specific backends.

**Comment by tkem** [comment]
Strictly speaking, Remotedy is not an MPD client but uses the Mopidy HTTP interface, so it belongs in a category with other Web clients.

**Comment by rawdlite** [comment]
You are absolutely right.
Yet i think this might not be the most helpful categorization to a user searching for a suitable client.
I imagine the first question to:
- Does it run on my phone or tablet
- Do i need a web browser
- Does it run in a terminal

The question what kind of api the client uses is of secondary interest only.
This being said with the client categorization as it is Remotely is not an MPD Client.

**Comment by tkem** [comment]
@rawdlite: True, I'm also not happy with this kind of categorization from a user perspective.
I think the categories you suggested (App, Web, Cmdline, Desktop(?)) make more sense.

**Comment by jodal** [comment]
The docs no longer have their own list of clients. The mopidy-mpd README links to the lists on Wikipedia and in the MPD wiki, and it documents `command_blacklist`.
